app/models/question.rb

```diff
@@ -21,16 +21,10 @@ def poll_responses_for(user)
 
 protected
   def options_for_multichoice
-    if type == "MultiChoiceQuestion" and qcontent.length < 2
```

Comment:
This was from Sommers, but not much is changing, so all good

Reply:
Yeah, admittedly this isn't that major of a fix but it does allow flexibility for rare cases of having only one answer choice